source/stylesheets/_refills.scss

```diff
@@ -0,0 +1,16 @@
+.refill {
+  border-top: 2px solid $base-border-color;
+  margin-bottom: $base-spacing * 6;
+  padding-top: $base-spacing;
+}
+
+.refill-wrapper {
+  align-items: center;
+  background-color: tint($base-border-color, 90%);
+  border: $base-border;
+  display: flex;
+  justify-content: space-around;
+  margin-top: $base-spacing;
+  min-height: $base-spacing * 12;
```

Comment:
Generally speaking I am not a fan of saying one must use variables for all units all the time. It ends up in a scenario where you just remake rems under a different label.

also saying 'lets add more variables' does not seem like a solution to me.